yet-another-docker-plugin/src/main/resources/com/github/kostyasha/yad/launcher/DockerComputerJNLPLauncher/help-jenkinsUrl.html

```diff
@@ -0,0 +1,3 @@
+<div>
+If you need your slaves to connect to a different jenkins server url.
```

Comment:
tunnel btw is other option, but could be added later and i don't remember how it works.